Assets/General/Scripts/Story/StoryManager.cs

```diff
@@ -50,6 +53,28 @@ private void DefineSingleton()
         }
     }
 
+    /// <summary>
+    /// THIS IS PURELY FOR TESTING PURPOSES
+    /// IF YOU ARE NOT CODE REVIEWING THIS TASK AND I DIDNT DELETE IT YOU CAN DIRECT ALL HATE TOWARDS RYAN :D
```

Comment:
hate --> ryan
lol just remember to delete

Reply:
or you could add a #IF UNITY ENGINE or # IF DEV BUILD so we can take advantage of this for testing